debian/ostree.install

```diff
@@ -1,5 +1,10 @@
+etc/dracut.conf.d/ostree.conf
 etc/ostree
+lib/systemd/system/ostree*.service
```

Comment:
It looks ostree is now using a systemd generator to handle mounting of `/var` during boot, so `lib/systemd/system-generators/ostree-system-generator` will also be needed here.